src/wp-includes/class-wp-script-modules.php

```diff
@@ -262,14 +272,53 @@ public function print_import_map() {
 	 *
 	 * @since 6.5.0
 	 *
-	 * @return array Array with an `imports` key mapping to an array of script module identifiers and their respective
-	 *               URLs, including the version query.
+	 * @global WP_Dependencies $wp_scripts
+	 *
+	 * @return array Array with an `imports` key mapping to an array of script
+	 *               module identifiers and their respective URLs, including
+	 *               the version query.
 	 */
 	private function get_import_map(): array {
+		global $wp_scripts;
+
 		$imports = array();
-		foreach ( $this->get_dependencies( array_keys( $this->get_marked_for_enqueue() ) ) as $id => $script_module ) {
-			$imports[ $id ] = $this->get_src( $id );
+
+		$classic_script_dependencies = array();
+		if ( $wp_scripts instanceof WP_Scripts ) {
+			foreach ( $wp_scripts->registered as $dependency ) {
```

Comment:
This is similar to what `wp_dependencies_unique_hosts` does:

https://github.com/WordPress/wordpress-develop/blob/ef7606073b8a016c5e32e230e50685ec62b3e7d2/src/wp-includes/general-template.php#L3724-L3726

I'm not confident this is the most optimal way to get all classic scripts in the enqueued dependency graph so would appreciate scrutiny or domain knowledge on that.